config/mail.php

```diff
@@ -36,7 +36,7 @@
     'mailers' => [
         'smtp' => [
             'transport' => 'smtp',
-            'host' => env('MAIL_HOST', 'smtp.mailgun.org'),
```

Comment:
pentru testare locala poti folosi mailCatcher sa nu trebuiasca sa trimiti mailuri reale. Ti le prinde in aplicatie pentur test.